sections/semantics-forms.include

```diff
@@ -7166,7 +7166,7 @@ You cannot submit this form when the field is incorrect.</samp></pre>
   user to pick an <{option}> element in its <a>list
   of options</a> that is itself not disabled. Upon
   this <{option}> element being <dfn>picked</dfn> (either
-  through a click, or through unfocusing the element after changing its value, or through any other mechanism), and before the
+  through a click, or through any other mechanism), and before the
```

Comment:
I suggest adding "keyboard-based mechanism" or something that means the same but doesn't sound as bad...